### PR TITLE
Clarified the Java 8+ requirement.

### PR DESCRIPTION
**Pull request by jwegne**
All implementations are fine (from Oracle, OpenJDK, Amazon, ...)

### CODE REVIEW

**Comment by goshev** [comment]
I don't see what was not clear.

**Comment by deeplook** [comment]
@goshev Do you see that it's clearer now?

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ![XYZ Hub](xyz.svg)
----
+
```
Comment (goshev) [inline README.md:2]: Could we keep the horizontal line after the logo?

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ XYZ Hub uses [GeoJSON](https://tools.ietf.org/html/rfc79460) as the main geospat
 
 # Prerequisites
 
- * Java 8+
+ * Java 8+ (e.g. Oracle Java 8+, OpenJDK 8+, Amazon Coretto 8+, ...)
```
Comment (goshev) [inline README.md:28]: Correct would be simply _Java 8.x_

File: README.md (hunk 1 above)
Comment (jwegne) [reply README.md]: Sorry, reverted the change.

File: README.md (hunk 2 above)
Comment (jwegne) [reply README.md]: Good point, changed to Java 8.x.

**Review by goshev — APPROVED** [review]

**Pull request merged**